tests/test_oracle_controller.py

```diff
@@ -36,16 +36,22 @@
 @pytest_asyncio.fixture(scope="module")
 async def contract_classes():
     account_class = compile_starknet_files(
-        files=[ACCOUNT_CONTRACT_FILE], debug_info=True
+        files=[ACCOUNT_CONTRACT_FILE], debug_info=True, cairo_path=["contracts/src"]
     )
     publisher_registry_class = compile_starknet_files(
-        files=[PUBLISHER_REGISTRY_CONTRACT_FILE], debug_info=True
+        files=[PUBLISHER_REGISTRY_CONTRACT_FILE],
+        debug_info=True,
+        cairo_path=["contracts/src"],
     )
     oracle_controller_class = compile_starknet_files(
-        files=[ORACLE_CONTROLLER_CONTRACT_FILE], debug_info=True
+        files=[ORACLE_CONTROLLER_CONTRACT_FILE],
+        debug_info=True,
+        cairo_path=["contracts/src"],
     )
     oracle_implementation_class = compile_starknet_files(
-        files=[ORACLE_IMPLEMENTATION_CONTRACT_FILE], debug_info=True
+        files=[ORACLE_IMPLEMENTATION_CONTRACT_FILE],
+        debug_info=True,
+        cairo_path=["contracts/src"],
```

Comment:
define as global var and use across test files

Reply:
Is there a way to do this that doesn't include making a wrapper function for `compile_starknet_files`?